flutter_common/lib/data/results/benchmark_result.dart

```diff
@@ -34,25 +34,25 @@ class Accuracy {
 
 class BenchmarkLoadgenInfo {
   static const String _tagValidity = 'validity';
-  static const String _tagDuration = 'duration_ms';
+  static const String _tagDuration = 'duration';
```

Comment:
What happens with old results when we changed the tag here? Is it still correct?
For now, this may not be a big issue but later if the results are collected, we need a migration plan.

Reply:
Old results will fail to parse from JSON.

> but later if the results are collected, we need a migration plan

Yeah. We will need to be more careful with the JSON structure and create migration plans after the next release.

Reply:
If the `tag` does not match, the JSON file should still be parsed, but the unmatched `tag` will be empty. This should be the default behavior instead of dropping the old results and create a new JSON file. I don't know yet how to do that Dart (but did that in other languages) so we can create an issue for this.

Reply:
https://github.com/mlcommons/mobile_app_open/issues/518#issuecomment-1250741946